docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc

```diff
@@ -0,0 +1,77 @@
+= Setting up a GKE cluster
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+This page walks you through the important steps to get Cloudflow up and running. For more detailed instructions on how to use or build your own Cloudflow installer, refer to the https://github.com/lightbend/cloudflow/tree/master/installer[installer section] of the GitHub repo.
+
+To create a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system by selecting the latest release branch or download the latest release file from https://github.com/lightbend/cloudflow/releases[here], go to the `installer/` folder and follow the instructions below.
+
+* Run `gcloud init` if you are creating a GKE cluster and this is your first time doing so. This command initializes the gcloud system on your local and performs necessary settings of gcloud credentials. You can have a look at the details in the https://cloud.google.com/sdk/gcloud/reference/init[GKE documentation].
+
+NOTE:: This can be done once you have a valid Google account and have access to a project set up for you by the administrator.
+
+* Create a Kubernetes cluster with a specific name on a cloud provider of your choosing. We provide cluster launching scripts for GKE, AKS and EKS. `./create-cluster-[gke|aks|eks].sh <cluster-name>`
+* Make sure your cluster has at least appropriate storage class that offers ReadWriteMany (RWM) capability. Most cloud providers have their preferred storage class for that purpose (e.g. AzureFile on AKS and EFS CSI on EKS). We also provide a script `./install_nfs.sh` that you can run to install NFS that will satisfy the RWM characteristic.
+* Go to the Github release page for your chosen Cloudflow version and download the bootstrap script. It's in the list of assets. For example, this is the https://github.com/lightbend/cloudflow/releases/tag/v1.3.3[page] for Cloudflow 1.3.3.
+* Run the bootstrap script to install Cloudflow.
+
+This will take you through the process of installing Cloudflow in the cluster. In course of this it will ask a few questions regarding the storage class to be installed. The following should be the response to these:
```

Comment:
```suggestion
This will take you through the process of installing Cloudflow in the cluster. In the course of its execution, it will ask a few questions regarding the storage class to be installed. 
The following provides guidance on how to respond to these questions:
```